banish/models.py

```diff
@@ -46,11 +46,14 @@ class Banishment(models.Model):
 
     condition = models.CharField(
         max_length=255,
-        help_text='Some descriptive text goes here'
+        help_text='Enter an IP to ban/whitelist, or a User Agent string to ban'
     )
 
     def __unicode__(self):
-        return "Banished %s %s " % (self.type, self.condition)
+        if self.type == 'ip-address-whitelist':
```

Comment:
@jamonation looks like this condition will always give False, because Banish.type cannot contain 'ip-address-whitelist'. If so, lets remove it